setup.cfg

```diff
@@ -1,2 +1,5 @@
 [aliases]
 test=pytest
+
+[tool:pytest]
+addopts = -p no:warnings
```

Comment:
Why not? What warnings are there?

Reply:
I added it cause the following warning shows up originating from the jinja2 template language:

```
/lancet-dev/lib/python3.7/site-packages/jinja2/utils.py:485
/lancet-dev/lib/python3.7/site-packages/jinja2/utils.py:485: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import MutableMapping

/lancet-dev/lib/python3.7/site-packages/jinja2/runtime.py:318
/lancet-dev/lib/python3.7/site-packages/jinja2/runtime.py:318: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import Mapping

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

Originating from https://github.com/pallets/jinja/pull/867

Reply:
Also added that as a comment

Reply:
I'd rather not hide warnings. You should be able to disable that specific warning (including the specific location).

Reply:
Adapted :)